agent/structs/structs.go

```diff
@@ -482,6 +482,15 @@ type HealthCheck struct {
 	ServiceName string        // optional service name
 	ServiceTags []string      // optional service tags
 
+	HTTP                           string               `json:",omitempty"`
```

Comment:
Since there are so many keys here and this is likely to grow a little, how about making a `Definition` sub-structure in here?